### ISSUE TITLE
Uncontrolled refresh sync causing GPU to run at MAX

### ISSUE THREAD

**Issue by Pythonic-Rainbow** [KaptainWutax/MemoryTester#4]
**Describe the Bug**
High CPU and GPU usage even on decent hardware

**To Reproduce**
Simply launch the game with the mod , and check Task Manager.

**Expected Behavior**
Renders at V-Sync(or static 30fps), greatly reduces CPU and GPU usage

**Screenshots**
![image](https://user-images.githubusercontent.com/25037295/55467640-4a848e00-5634-11e9-904e-1380b9aeb25c.png)
![image](https://user-images.githubusercontent.com/25037295/55467803-9cc5af00-5634-11e9-90fc-aaaa7b81b869.png)

**Technical (please complete the following information):**
 - Operating System : e.g. Windows 10
 - MemoryTester Version : 0.4.2
 - Minecraft Version : 1.12.2
 - Forge Version (if applicable) : b2824

**Comment by KaptainWutax** [comment]
Yep thanks, I know where the issue is and I'll fix as soon as I can.
